js/app/controllers/editorcontroller.js

```diff
@@ -123,8 +123,8 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 
 		$scope.keypress = function(event) {
 			var code = event.keyCode || event.which;
-			if(event.ctrlKey===true && code === 13) {
-				$scope.$broadcast('save-contents');
+			if((event.metaKey===true || event.ctrlKey===true) && code === 13) {
```

Comment:
`if(event.metaKey===true && code === 13) {` should be just fine.

Meta-Key is true for CTRL on Windows / Linux and CMD on Mac :)
https://api.jquery.com/event.metakey/

Reply:
and please add spaces on the left side: `event.metaKey === true`, not `event.metaKey===true `

Reply:
Sorry, it doesn't map to CTRL on Windows, it maps to the windows key.
My fault